Source/WTF/wtf/CheckedRef.h

```diff
@@ -278,23 +278,23 @@ template <typename StorageType, typename PtrCounterType, DefaultedOperatorEqual
 
     ~CanMakeCheckedPtrBase() = default;
 
-    PtrCounterType ptrCount() const { return m_count; }
-    void incrementPtrCount() const { ++m_count; }
-    ALWAYS_INLINE void decrementPtrCount() const
+    PtrCounterType checkedPtrCount() const { return m_checkedPtrCount; }
+    void incrementCheckedPtrCount() const { ++m_checkedPtrCount; }
```

Comment:
Hm... this will require clang side change.

Reply:
How will we deal with this?

Reply:
Oops. I had missed this comment. I have ping'd Ryosuke on Slack to see if I need to revert or not.